cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -245,7 +245,7 @@ func (d *driver) nodePrepareResource(ctx context.Context, claim *resourceapi.Res
 	devs, err := d.state.Prepare(ctx, claim)
 	if err != nil {
 		res := kubeletplugin.PrepareResult{
-			Err: fmt.Errorf("error preparing devices for claim %v: %w", claim.UID, err),
+			Err: fmt.Errorf("error preparing devices for claim %s/%s:%s: %w", claim.Namespace, claim.Name, claim.UID, err),
```

Comment:
Maybe we can have a convenience method for this normalized string representation, just like we have for claimref.